python_on_whales/components/context/cli_wrapper.py

```diff
@@ -81,9 +81,40 @@ def use(self) -> None:
 
 
 class ContextCLI(DockerCLICaller):
-    def create(self):
-        """Not yet implemented"""
-        raise NotImplementedError
+    def create(
+            self,
+            context: str,
+            host: str = None,
+            description: str = None,
+            from_: str = None,
+            from_host: ValidContext = None
+    ):
+        """Creates a new context
+
+        # Arguments
+            context: name of the context to create
+            host: uri of the host to access
+            description: description of the context
+            from_: create context from a named context
+            from_host: use endpoint configuration from existing named context
+        """
+        existing_contexts = [str(x) for x in self.list()]
+        if context not in existing_contexts:
+            full_cmd = self.docker_cmd + ["context", "create"]
+            if from_:
+                full_cmd += ["--from", from_]
+            else:
+                full_cmd += ["--docker"]
+                if from_host:
+                    if from_host in existing_contexts:
+                        full_cmd += [f"from={from_host}"]
+                else:
+                    full_cmd += [f"host={host}"]
+                if description:
+                    full_cmd += [f"--description={description}"]
```

Comment:
For this kind of situation, can you use the methods already declared in the class? You shouldn't have to manipulate lists at all here, it should make it easier. You can take a look in `docker.container.run()` https://github.com/gabrieldemarmiesse/python-on-whales/blob/master/python_on_whales/components/container/cli_wrapper.py#L611 to have examples of such methods :) 
In this situation I think you can use `self.add_simple_arg()`.

Reply:
Will change that ones. Thank you and sorry that I am not really into your code.

Reply:
No worries!

Reply:
The `self.add_simple_arg()` is joining the arguments via whitespace, but I need it with an equal sign. Let me do an example:
`--description test` should become `--description=test`. Can you help me out here?

Reply:
I think whitespace is ok no? Docker usually accepts whitespace and equal.

Reply:
No actually it runs into an error:
`python_on_whales.exceptions.DockerException: The docker command executed was `/usr/bin/docker context create --docker host ssh://test@test.domain --description Python on whales testing context testpow``

Reply:
It was the combination of `--docker`, `host` and then `value`... will change it.